### DISCUSSION TITLE
[Question] Is there a plan to implement KSeF for invoices issued in Poland?

### DISCUSSION THREAD

**Discussion by dextras** [YetiForceCompany/YetiForceCRM#16795] [How to] [answered]
I am wondering if there is a plan to implement KSeF integration, even as a paid plugin, in the near future? 
In Poland it is going to be required to use that system from the beginning of 2024. In such scenario, using Invoice module from Yetiforce without the integration with KSeF becomes harder as invoices would have to be processed twice:
* in KSeF
* in Yetiforce.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
yes, we have it in the plans
